ApplicationCatalogGui/1.0.0/installer-manifest.json

```diff
@@ -0,0 +1,17 @@
+{
+    "wire-navigation": [
+        {
+            "navigation": {
+                "sales.application-catalog-gui": {
+                    "label": "Apps",
+                    "title": "Apps",
+                    "icon": "fa-archive",
+                    "bundle": "application-catalog-gui",
```

Comment:
we should not use long outdated name here if we can.
is this required to be called this way?

Reply:
I am not sure how the navigation builder works but I guess this property is required and as I understand it is a module name that is used during the building of routes.
Or I did not get your question.

Reply:
Sure, I mean the manifest could use new term and could be mapped to "old" one for applying.
But I can also see why keeping the old name might be easier for the tooling for now.
@stereomon What do you think?

Reply:
I'm ok to map "module" in manifest, but "bundle" in the generated xml

Please assume we are going to change the module name very very soon to `app-catalog-gui`.

Reply:
I'm also fine with changing the current `bundle` key with `module`. The strategy would then map `module` to `bundle` while building the new navigation.

Reply:
Does the position of the bundle key(tag) matter in the navigation data? E.g.:
```
    <before-main>
        <label>Before main</label>
        <title>Before main</title>
        <icon>fa-archive</icon>
        <controller>index</controller>
        <action>index</action>
        <pages>
            <before-main-nested>
                <label>Before main - nested</label>
                <title>Before main - nested</title>
                <icon>fa-archive</icon>
                <controller>index</controller>
                <action>index</action>
                <visible>0</visible>
                <bundle>before-main-nested</bundle>
            </before-main-nested>
        </pages>
        <bundle>before-main</bundle>
    </before-main>
```

Or should it be before the controller tag?

Reply:
order in XMLs do not matter. but for human readonability it can be nice to have it ordered semantically.

Reply:
ok, will add ordering

Reply:
As I see, Oleh added the correct order in commit from February 28 .